### PR TITLE
docs: validating webhook policies to limit number of replicas

### PR DESCRIPTION
**Pull request by Tomer20**
Added examples of `ValidatingAdmissionPolicy` to fail/deny `ScaledObject`s based on `maxReplicaCount`.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

File: content/docs/2.15/operate/admission-webhooks.md [hunk 1]
```diff
@@ -9,3 +9,95 @@ weight = 100
 By default, the admission webhooks are registered with `failurePolicy: Ignore`, this won't block the resources creation/update when the admission controller is not available. To ensure that the validation is always required and perform validation, setting `failurePolicy` to `Fail` is required.
 
 In particular, the admission webhooks for HPA ownership validation can be skipped by setting the annotation `validations.keda.sh/hpa-ownership` to `"false"`. Be cautious when doing so as it exposes the system to potential risks.
+
+Here is an example of a `ValidatingAdmissionPolicy` and its corresponding `ValidatingAdmissionPolicyBinding` to fail/deny `ScaledObject`s with more than 10 replicas in the `default` namespace:
```
Comment (zroubalik) [inline content/docs/2.15/operate/admission-webhooks.md:13]: could you add some title here, please?

For example: `## Custom Validations using ValidatingAdmissionPolicy`

And a short sentence introducing this section together with a link to kubernetes docs? (https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/)

**Review by zroubalik — COMMENTED** [review]
Thanks!

Could you please also add similar change to version 2.16?

File: content/docs/2.15/operate/admission-webhooks.md (hunk 1 above)
Comment (Tomer20) [reply content/docs/2.15/operate/admission-webhooks.md]: yes, totally makes sense. please see changes.

**Comment by Tomer20** [comment]
> Thanks!
> 
> Could you please also add similar change to version 2.16?

Thanks for the feedback. I added the docs for v2.16 as well.

**Review by zroubalik — APPROVED** [review]
Thank you!

**Pull request merged**